packages/kit/src/core/dev/index.js

```diff
@@ -167,8 +167,12 @@ class Watcher extends EventEmitter {
 				}
 			},
 			ssr: {
-				// @ts-expect-error - ssr is considered in alpha, so not yet exposed by Vite
-				noExternal: [...((vite_config.ssr && vite_config.ssr.noExternal) || []), ...svelte_packages]
+				noExternal: [
+					'@sveltejs/kit', // TODO: see https://github.com/vitejs/vite/issues/3953
+					// @ts-expect-error - ssr is considered in alpha, so not yet exposed by Vite
+					...((vite_config.ssr && vite_config.ssr.noExternal) || []),
```

Comment:
Could be shortened to `vite_config?.ssr?.noExternal ?? []`

Reply:
AFAIK it can't, because the supported Node versions don't support that syntax.

Reply:
Upsi, i just realized that we still have a dual dependency of node 12 & 14 🙈